src/pages/DatasetCatalog.js

```diff
@@ -506,17 +514,16 @@ class DatasetCatalog extends Component {
                             get(find(dacs, dac => { return dac.id === dataSet.dacId; }), 'name', '')
                           ]),
 
-                          td({ className: 'table-items cell-size ' + (!dataSet.active ? 'dataset-disabled' : '') }, [
-                            a({
-                              id: trIndex + '_linkdbGap',
-                              name: 'link_dbGap',
-                              href: get(find(dataSet.properties, p => { return p.propertyName === 'dbGAP'; }), 'propertyValue', ''),
-                              target: '_blank',
-                              className: (get(find(dataSet.properties, p => { return p.propertyName === 'dbGAP'; }), 'propertyValue', '').length > 0 ?
-                                'enabled' :
-                                'disabled')
-                            }, ['Link'])
-                          ]),
+                            td({ className: 'table-items cell-size ' + (!dataSet.active ? 'dataset-disabled' : '') },
```

Comment:
Outside the scope of this PR, but we should be looking to replace class names with inline styles when possible. Not required for this PR.